### PR TITLE
[Fix] refactor parse for figure

### PR DESCRIPTION
**Pull request by ranjabi**
## Story/Task

_Fix figure such as image, video, youtube, twitter, spotify, gallery_

## Summary

-   _Add figure attribute interface_
-   _Add attribute parser_
-   _Add caption and alt for image
_

## Test Plan
_Add figure element_

## Screenshot
image
image

### CODE REVIEW

File: src/pages/Detailpage/Render.tsx [hunk 1]
```diff
@@ -141,12 +141,22 @@ const Render = {
...
+                      'font-Bold'
+                  )
+                : undefined;
+            const description = node.childNodes[0].childNodes[2].childNodes[0]
+                ? renderText(
+                      node.childNodes[0].childNodes[2].childNodes[0]
+                          .childNodes[0],
+                      id
+                  )
+                : undefined;
```
Comment (IloveNooodles) [inline src/pages/Detailpage/Render.tsx:159]: ini mending undefined apa null aja

**Review by IloveNooodles — APPROVED** [review]
semua yg undefined mau diganti null aja ga? ada alesan spesifil kenapa make undefined?

File: src/pages/Detailpage/Render.tsx (hunk 1 above)
Comment (ranjabi) [reply src/pages/Detailpage/Render.tsx:159]: @grevicoc buat div yg product nih kak

Comment (IloveNooodles) [reply src/pages/Detailpage/Render.tsx:159]: leih ke semuanyaa sii banyak undefined gituu

**Pull request merged**